Makefile

```diff
@@ -9,7 +9,7 @@ install: Example/Podfile $(PROJECT).podspec
 	pod install --project-directory=Example
 
 lint:
-	pod lib lint --allow-warnings
+	pod lib lint --allow-warnings --use-libraries
```

Comment:
Adobe is still a static library. Shouldn't this flag still be required?

Reply:
Hmm not sure what you mean, the flag was added not removed

Reply:
Ah ok read the diff wrong.